kubernetes/patroni_k8s.yaml

```diff
@@ -120,3 +121,60 @@ type: Opaque
 data:
   superuser-password: emFsYW5kbw==
   replication-password: cmVwLXBhc3M=
+
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: patronidemo
+
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: patronidemo
+rules:
+- apiGroups:
+  - ""
+  resources:
+  - configmaps
+  verbs:
+  - create
+  - get
+  - list
+  - patch
+  - update
+  - watch
+- apiGroups:
+  - ""
+  resourceNames:
+  - patronidemo
+  resources:
+  - endpoints
+  verbs:
+  - get
+  - patch
+  - update
+- apiGroups:
+  - ""
+  resources:
+  - pods
+  verbs:
+  - get
+  - list
+  - patch
+  - update
+  - watch
```

Comment:
Patroni never watch on `Pods`

Reply:
It's more of an approach we in OpenShift take, that whenever you have list a watch is just there. That's also what's represented in the tool I've mentioned and why this is here.

Reply:
From a security pov having either or both is almost the same.